docs/blockchain/contracts/nft-contract/nft-actions/setprchsreq.a.staging.md

```diff
@@ -182,7 +182,11 @@ The logical operators' values are defined as
         * = [2305843009213693953, 3458764513820540930]
     - parameter value  ``` "group_restriction": [2305843009213693953, 3458764513820540930] ```
 
+## Integrating first hand purchase into your own solution
 
+We provide an example project that utilizes first hand purchase functionality to allow users to exchange Uniqs they have into other Uniqs. You can clone, fork or adopt this solution to your own needs.
```

Comment:
We offer a sample project showcasing the utilization of firsthand purchase functionality. This enables users to exchange their Uniqs for other Uniqs. Feel free to clone, fork, or customize this solution according to your requirements.